src/component/calculator.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import './calculator.css';
```

Comment:
- [ ] There is no `calculator.css` file. Kindly remove this line.

Reply:
@medaminedev66  thank you for the feedback, I have removed the unused files . 

![e49](https://user-images.githubusercontent.com/40554384/164144922-f00cb72a-fe39-4203-94df-d3172032f876.PNG)

Reply:
@medaminedev66  Thank you i have also removed the line for the import './calculator.css';
![e50](https://user-images.githubusercontent.com/40554384/164145104-1be6f852-6834-43af-bbe7-f1b06384e56b.PNG)